angular_bind_polymer.js

```diff
@@ -2,6 +2,7 @@ angular.module('eee-c.angularBindPolymer', []).
 directive('bindPolymer', function() {
   return {
     restrict: 'A',
+    scope: {}, /* Scope added here */
```

Comment:
Pardon me if my Angular knowledge is not 100% accurate, but doesn't this restrict observers to nothing? That is, without at least a single scope entry, won't this directive observe nothing?

I'm still not clear why you are trying to [restrict scope](https://docs.angularjs.org/guide/directive#isolating-the-scope-of-a-directive) like this. Can you update the pull request so that it better approximates what you are trying to accomplish?

FWIW this might not work with the `=` from the original ticket. I am  relying on double-curly braces to identify angular properties in the Polymer attribute. The [`@` approach](http://stackoverflow.com/questions/14050195/what-is-the-difference-between-and-in-directive-scope) might  work, but I couldn't quite get it working with the `<x-double>` example in this repo.